javascript-bank/account.js

```diff
@@ -0,0 +1,47 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (amount <= 0 || typeof amount === 'string' || !amount) {
+    return false;
+  } else {
+    var newDeposit = new Transaction('deposit', amount);
+    this.transactions.push(newDeposit);
+    return true;
+  }
+};
+
+Account.prototype.withdraw = function (amount) {
+  if (amount <= 0 || typeof amount === 'string' || !amount) {
+    return false;
+  } else {
+    var newWithdrawal = new Transaction('withdrawal', amount);
+    this.transactions.push(newWithdrawal);
+    return true;
+  }
+};
+
+Account.prototype.getBalance = function () {
+  if (this.transactions.length === 0) {
```

Comment:
This conditional is unnecessary. If the array is empty, the loop below would never start, which leaves you in the same situation 👍